app/pom.xml

```diff
@@ -65,9 +65,9 @@
         </configuration>
         <executions>
           <execution>
-            <id>dockerBuild</id>
```

Comment:
I presume you're doing this so you can use the `build` goal? If so, could you also remove the commented out TODO execution below this one?